plugins/candela/plugin_tests/candelaSpec.js

```diff
@@ -19,6 +19,10 @@ describe('Test the candela UI.', function () {
         });
         girderTest.waitForLoad();
```

Comment:
We should remove the `waitForLoad` here as well as the one above, and we can combine the previous two `runs` blocks into one since we don't have any asynchronous action to wait for between them.